test/testio.cpp

```diff
@@ -4804,20 +4805,28 @@ class TestIO : public TestFixture {
     }
 
     void testPrintfParenthesis() { // #8489
-        check("void f(int a) {\n"
-              "    printf(\"%f\", (a >> 24) & 0xff);\n"
-              "}");
-        ASSERT_EQUALS("[test.cpp:2]: (warning) %f in format string (no. 1) requires 'double' but the argument type is 'signed int'.\n", errout.str());
-
-        check("void f(int a) {\n"
-              "    printf(\"%f\", 0xff & (a >> 24));\n"
-              "}");
-        ASSERT_EQUALS("[test.cpp:2]: (warning) %f in format string (no. 1) requires 'double' but the argument type is 'signed int'.\n", errout.str());
+      check("void f(int a) {\n"
```

Comment:
did you just reformat the code here? It would be nice to revert back to the original code then.

Reply:
See response below.